src/main/java/com/revature/_1811_nov27_wvu/icebox/services/FileServiceImp.java

```diff
@@ -0,0 +1,37 @@
+package com.revature._1811_nov27_wvu.icebox.services;
+
+import java.util.Set;
+
+import com.revature._1811_nov27_wvu.icebox.dao.FileDAO;
+import com.revature._1811_nov27_wvu.icebox.dao.FileHibernate;
+import com.revature._1811_nov27_wvu.icebox.entity.File;
+
+public class FileServiceImp implements FileService {
+	private FileDAO fd = new FileHibernate();
```

Comment:
FileDao should be using the `@Autowired` annotation and not directly creating the object.